### DISCUSSION TITLE
为什么找不到net包啊

### DISCUSSION THREAD

**Discussion by wdskdl** [WPFDevelopersOrg/WPFDevelopers#80] [General]
为什么找不到net包啊

**Comment by yanjinhuagood** [comment]
Nuget需要勾选预览包

**Reply by wdskdl** [reply]
谢谢，已经可以了，请问有什么可以让我学习开发的文档嘛

**Reply by yanjinhuagood** [reply]
1.下载[releases](https://github.com/WPFDevelopersOrg/WPFDevelopers/tags)可以查看代码示例
2.看下[Wiki](https://github.com/WPFDevelopersOrg/WPFDevelopers/wiki)，Wiki不怎么全面

**Reply by wdskdl** [reply]
非常感谢

**Reply by wdskdl** [reply]
如果我想让message一秒之后自动关闭我应该怎么做呢

**Reply by wdskdl** [reply]
暂时好像不行，是十秒自动关闭

**Reply by yanjinhuagood** [reply]
暂时不支持设置关闭时间，你可以提一个[issue](https://github.com/WPFDevelopersOrg/WPFDevelopers/issues/new)我们来增加。
